packages/cozy-harvest-lib/src/components/AccountForm/index.jsx

```diff
@@ -190,6 +190,9 @@ export class AccountForm extends PureComponent {
             ref={element => {
               container = element
             }}
+            // Hacking manually the style here, waiting for Cozy-UI to remove
+            // top margin on Field's label elements.
```

Comment:
I think it is very useful to add a link to an issue, this way when we see this text,  we can check the issue to see if it is resolved.

Reply:
Done :thumb